recipes-bsp/rpi-boot-setting/files/boot.cmd

```diff
@@ -0,0 +1,4 @@
+fatload mmc 0 ${kernel_addr_r} Image
+fatload mmc 0 ${fdt_addr_r} bcm2837-rpi-3-b-plus.dtb
```

Comment:
It'd be nice dtb file name is not immediate value to support other raspberry pi board in the future.

Reply:
We would have to add a scirpt to do_prepare_build() in the recipe in order to insert any line to the boot.cmd script to ahieve what you metioned. Is it enough to work on that when we support a new board?

Reply:
Yes. That's fine.